lib/YaST/EFItools.pm

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package YaST::EFItools;
+
+use strict;
+use warnings;
+use Exporter 'import';
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+our @EXPORT = qw(
+  check_secure_boot_status
+);
+
+use constant MIN_OCTALS => 5;
+
+=head1 tools for EFI related stuff
```

Comment:
neapicking, "stuff" is too colloquial, perhaps s/stuff/operations/

Reply:
Changed.